AlertaDengue/environment.yml

```diff
@@ -28,9 +29,6 @@ dependencies:
 - geopy
 - geopandas
 - sh
-- cufflinks-py
 - ipython
 - sqlparse
-- pip
-- pip:
-    - plotly==2.5.1
\ No newline at end of file
+- plotly=4.0.0rc1
```

Comment:
Shouldn't this be `plotly==4.0.0rc1` ?

Reply:
IIRC `=` works for conda packages ... and for packages inside pip section it should be `==`